### ISSUE TITLE
Weird Beta outputs

### ISSUE THREAD

**Issue by Darenar** [ragulpr/wtte-rnn#64]
Hello!
I am implementing WTTE RNN on self-collected data from a company, where I have a censorship rate around 50 %. 
There are 30 000 sequences with 500 timesteps.
The thing is that my TTE could be very large (up to 400 days), though predicted beta for each time step in train/valid/test is less than 1, which in turn seems weird for me (coz I can't even calculate mode on the resulted distribution - its 0 everywhere). Nevertheless during training loss function decreases over 15 epochs approximately. My data is normalised and censorship indicator + TTE seems to be correct.
Additionally, I tried fitting model only on non-censored sequences, and the result is still the same.
May be you have any ideas how is it possible could be?

Thanks!
